tea.py

```diff
@@ -0,0 +1,674 @@
+import pybamm
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+class Tea(pybamm.BaseModel):
+    """"""
+
+    def __init__(self, parameter_values=None):
+        self.parameter_values = parameter_values
+        if self.parameter_values is None:
+            print(
+                "Warning: Missing parameter values, please supply parameter values to Tea()"
+            )
+        super().__init__()
+
+    def plot_ocvs(self, parameter_values=None):
+        # plot ocv curves, average values against gravimetric capacity of cam and
+        # what if two different parameter sets are used? if, try, except?
+        # in case it is called two times after each other, the second time it will not work
+        # the solution is to call it the second time
+        parameter_values = self.parameter_values
+        try:
+            U_n = parameter_values["Negative electrode OCP [V]"]
+            U_p = parameter_values["Positive electrode OCP [V]"]
+        except KeyError:
+            print("Error: Missing required parameters")
+            return
+
+        x0, x100, y100, y0 = pybamm.lithium_ion.get_min_max_stoichiometries(
```

Comment:
you re-do this calculation a few times, so you do it once and store the result as an attribute